homeassistant/components/flexit_bacnet/sensor.py

```diff
@@ -0,0 +1,252 @@
+        translation_key="supply_air_fan_rpm",
+        value_fn=lambda data: data.supply_air_fan_rpm,
+    ),
+    FlexitSensorEntityDescription(
+        key="exhaust_air_fan_control_signal",
+        state_class=SensorStateClass.MEASUREMENT,
+        translation_key="exhaust_air_fan_control_signal",
+        value_fn=lambda data: data.exhaust_air_fan_control_signal,
+        native_unit_of_measurement=PERCENTAGE,
+    ),
+    FlexitSensorEntityDescription(
+        key="exhaust_air_fan_rpm",
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=REVOLUTIONS_PER_MINUTE,
+        translation_key="exhaust_air_fan_rpm",
+        value_fn=lambda data: data.exhaust_air_fan_rpm,
+    ),
+    # This does not make sense to be a sensor, since it's value is always the same, i.e. 0.8 kW
+    # Could this be added as some form of extra attribute somewhere?
+    # FlexitSensorEntityDescription(
+    #     key="electric_heater_nominal_power",
+    #     device_class=SensorDeviceClass.POWER,
+    #     native_unit_of_measurement=UnitOfPower.KILO_WATT,
+    #     translation_key="electric_heater_nominal_power",
+    #     value_fn=lambda data: data.electric_heater_nominal_power,
+    #     suggested_display_precision=3,
+    # ),
+    FlexitSensorEntityDescription(
+        key="electric_heater_power",
+        device_class=SensorDeviceClass.POWER,
+        native_unit_of_measurement=UnitOfPower.KILO_WATT,
+        translation_key="electric_heater_power",
+        value_fn=lambda data: data.electric_heater_power,
+        suggested_display_precision=3,
+    ),
+    FlexitSensorEntityDescription(
+        key="air_filter_operating_time",
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        suggested_display_precision=0,
+        suggested_unit_of_measurement=UnitOfTime.HOURS,
+        native_unit_of_measurement=UnitOfTime.HOURS,
+        translation_key="air_filter_operating_time",
+        value_fn=lambda data: data.air_filter_operating_time,
+        # should we have a last_reset?
+    ),
+    FlexitSensorEntityDescription(
+        key="heat_exchanger_efficiency",
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=PERCENTAGE,
+        translation_key="heat_exchanger_efficiency",
+        value_fn=lambda data: data.heat_exchanger_efficiency,
+    ),
+    FlexitSensorEntityDescription(
+        key="heat_exchanger_speed",
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=PERCENTAGE,
+        translation_key="heat_exchanger_speed",
+        value_fn=lambda data: data.heat_exchanger_speed,
+    ),
+)
+
+# These attributes are already in the climate entity
+#   operation_mode
+#   ventilation_mode
+
+# Comfort button state: shows true or false if the hvac is in mode HOME or AWAY, and is probably not needed as a sensor
+# room humidity 1,2,3: gives readings if extra wireless hardware is installed (so not needed)
```

Comment:
Can we detect if extra hardware is installed? You can add more entities to add based on hardware type

Reply:
Detection will come in the future, when we know the difference between the different models.